content/en/istio-ca-certs-integrations/cert-manager-integration/_index.md

```diff
@@ -188,13 +184,170 @@ patches:
 
 The installation should complete, and the Istio control plane should become in a
 ready state.
-
 ```sh
-$ kubectl get pods -n istio-system
-NAME                                   READY   STATUS    RESTARTS   AGE
-istio-ingressgateway-9ddfd475f-47ssj   1/1     Running   0          45s
-istiod-6ff8d84d66-tvdl2                1/1     Running   0          71s
+kubectl get pods -n istio-system
 ```
 
+## Test mTLS
+
 All workload certificates will now be requested through cert-manager using the
-configured Issuer.
+configured Issuer. Let's run an example workload to test mTLS. First, create
+a namespace and configure it for automatic sidecar injection:
+```sh
+kubectl create ns foo
+kubectl label ns/foo istio-injection=enabled
+```
+
+Run the sample sleep and httpbin workloads.
+```sh
+ISTIO_VERSION=$(getmesh version | grep "control plane" | cut -d " " -f 4 | cut -d "." -f 1,2)
```

Comment:
I think relying on the shell string manipulation is fragile.
We can just make it such `ISTIO_VERSIO="1.9.1" # change to your own version`.

Reply:
Ack